### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by Mohamed209** [omar-florez/scratch_mlp#8]
i am little confused , it is not possible to use RMSE loss function with classification problem , you may use cross entropy loss
